File: Models/ReturnModel.cs
using System.Collections.Generic;

namespace Models
{
    public class ReturnModel<T>
    {
        private List<T> data;
        private int status;
        private string message;

        public ReturnModel(List<T> data, int status, string message) 
        {
            this.data = data;
            this.status = status;
            this.message = message;
        }
    }
}

File: ClientServer/client_server/src/Models/ReturnModel.ts
export class ReturnModel{
    public data!: Array<any>;
    public status!: number;
    public message!: string;

    constructor(data:Array<any>, status:number, message:string)
    {
        this.data = data;
        this.status = status;
        this.message = message;
    }

    public setMessage(message: string){
        this.message = message;
    }

    public Set(data:Array<any>, status:number, message:string)
    {
        this.data = data;
        this.status = status;
        this.message = message;
    }
}

